### PR TITLE
essay on globally abundant water and other things!

### PR DESCRIPTION
**Pull request by intldds**
hope you enjoy it, I had a lot of fun writing this one :)

### CODE REVIEW

File: dialogue/local-abundance.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+P: Sure, go ahead.
+
+S: How is it possible that your name is Play, yet I sense some sadness and fatigue in your tone?
+
+P: That’s a great question – it’s quite counterintuitive, actually. Just because my name is Play, people assume my life is like a blossoming garden. I don’t see it that way. I look at myself more like water.
+
+S: Water?
+
+P: Yes, water. There’s something about water that makes it so special and unique. It’s hard to describe it with words – it’s more of a feeling, you know? Our [terminology](https://jzhao.xyz/thoughts/terminology/) isn’t quite there yet. Nevertheless, I’ll explain to you why water is something so close to my heart. Growing up, I used to take myself very seriously. It was definitely not fun and lasted for a couple of years. My mom eventually took me to this psychologist who diagnosed me with something called “atelophobia”. What do psychologists know about life anyway? I’ve had to figure out on my own how to get out of that bubble.
```
Comment (jackyzha0) [inline dialogue/local-abundance.md:15]: Would love either more description about "atelophobia" or at least a footnote about it! I'm curious about how this affects P's day to day

File: dialogue/local-abundance.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+S: How did you become like water?
+
+P: You are a very curious person, I’m starting to like you. Growing up with the Internet, I started to understand I could reverse engineer my life and become whatever I wanted to. Once I realized that there’s not really any limitation, I decided to become like water.
+
+S: That’s a peculiar choice.
+
+P: I’ve always looked at life as a choice between intellectual honesty and ego. Intellectual honesty is just a fancy way to say “play”. Ego is everything we do to seek status. Just by observing adults, I realized many of them are just seeking status. Honestly, they look miserable; I’m quite good at spotting sad people. Kids, on the other hand, have this playful view of the world. It’s something very intrinsic and doesn't come from ego. Ego is the worst thing in life, it destroys everything. People usually develop an ego when they start compromising on things they love in order to do things they don’t want to do. Eventually, this “ego” thing starts growing, and people start pretending to love what they do. That’s why I became water. Water has no ego and seeks no [status](https://twitter.com/NavalismHQ/status/1557575533730430976). Water flows.
```
Comment (jackyzha0) [inline dialogue/local-abundance.md]: yes! Intellectual 'play' turns out to have epistemic functionality as well https://jzhao.xyz/thoughts/Jestermaxxing

File: dialogue/local-abundance.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+S: I agree with you, ego is the worst thing we can have. However, you mentioned earlier that people assume your life is like a blossoming garden due to your lusory attitude. Isn’t a blossoming garden more exciting than water?
+
+P: I see my life more like dirt, as opposed to a blossoming garden. Dirt gives you the opportunity to grow new things, it makes room for surprises. I’m more interested in the process than in the outcome. That’s why my name is Play.
```
Comment (jackyzha0) [inline dialogue/local-abundance.md:35]: How does dirt differ from water here? It seems like you use dirt here to refer to the characteristic of being useful towards growth and water to refer to the ability to not attach yourself to perfection and to just flow.

Doesn't growth require persistence and attachment though? Detach a plant from its dirt and it will quickly die

File: dialogue/local-abundance.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+S: But still, I have one question left. Since you became like water, have you found seeds to nurture?
+
+P: I’m starting to find them. They were always here, but somehow I was distracted. They say “think globally, act locally”, right? I always believed that but, due to the Internet, we can act globally in a permissionless way. For quite some time, I was looking for places of local abundance. It’s like I was looking for a Schelling Point for all playful actors. We are all unpaid actors in this game of life. Recently, I found out that the Internet is a beautiful Schelling Point for us. It made local abundance a global place. We’re all together in this campfire now! We can play, we can build, we can create, we can discuss ideas.
```
Comment (jackyzha0) [inline dialogue/local-abundance.md]: Would love a link for Schelling Point

File: package-lock.json [hunk 5]
```diff
@@ -1,6416 +1,26 @@
 {
   "name": "play",
-  "lockfileVersion": 2,
   "requires": true,
-  "packages": {
```
Comment (jackyzha0) [inline package-lock.json]: is it possible to revert this change? Would prefer to keep the existing package-lock :)

**Review by jackyzha0 — COMMENTED** [review]
First off, thank you @intldds for this beautiful essay :)) Just left a few comments that I think would make for interesting discussion points/revisions but also happy to merge after you address/respond to the comments

File: dialogue/local-abundance.md (hunk 1 above)
Comment (intldds) [reply dialogue/local-abundance.md:15]: I've added a little description about atelophobia in the dialogue : )

File: dialogue/local-abundance.md (hunk 2 above)
Comment (intldds) [reply dialogue/local-abundance.md]: wow, this is amazing -- I haven't read it before! I've added your link. Intellectual playfulness is a really great concept, it really fits in with what I was trying to say!

File: dialogue/local-abundance.md (hunk 3 above)
Comment (intldds) [reply dialogue/local-abundance.md:35]: Yes, you are absolutely right! I've used dirt as a metaphor for growth/being useful and water as a metaphor for non-attachment; it's a bit counterintuitive because growth requires attachment like you said! That's why I started the dialogue with P saying "That’s a great question – it’s quite counterintuitive, actually". There's this internal battle in P's life between attachment and letting go. I didn't expand much on this and kind of let it open to interpretation -- you understood the metaphor really well. 

Nevertheless, I've added a little dialogue to expand on this. I've quoted you directly "Detach a plant from its dirt and it will quickly die", I think that sentence is amazing lol

File: dialogue/local-abundance.md (hunk 4 above)
Comment (intldds) [reply dialogue/local-abundance.md]: link added! (from wikipedia, it was the most credible-neutral source)

File: package-lock.json (hunk 5 above)
Comment (intldds) [reply package-lock.json]: upss, sorry! reverted back to the original package-lock :)

**Comment by intldds** [comment]
After reading your comments, I've also added a commit changing the title of the essay to "On playgrounds and balance". "Playgrounds" are meant to represent the Schelling Points for "intellectual players" (i.e. people who pursue intellectual playfulness). "Balance" is meant to represent the internal battle in P's life between non-attachment (flowing) and commitment (attachment to growth, desire to make impact). I think this title encapsulates a bit better some of the ideas presented, hope you like the new one @jackyzha0  :)

**Review by jackyzha0 — APPROVED** [review]
